include/spvgen.h

```diff
@@ -117,6 +117,9 @@ enum SpvGenStage : uint32_t
     SpvGenNativeStageCount = SpvGenStageCompute + 1,
 };
 
+void internalInit();
+void internalFinal();
```

Comment:
The new functions are in the global namespace. I'm worried this can cause name collision. Can we move them to some namespace or give them less generic names?

Reply:
I move them to `.cpp`. Thanks.